tests/commands_test.py

```diff
@@ -1186,7 +1183,7 @@ def test_delta_cmd(memcache_socket: MemcacheSocket, cache_client: CacheClient) -
 
     cache_client.delta(key=Key("foo"), delta=1, refresh_ttl=60, no_reply=True)
     memcache_socket.sendall.assert_called_once_with(
-        b"ma foo q D1 T60\r\n", with_noop=True
+        b"ma foo q T60 D1\r\n", with_noop=True
```

Comment:
Looks like order on the wire doesn't matter for the flags, nothing here specifies a required order: https://github.com/memcached/memcached/blob/master/doc/protocol.txt#L116 so this seems ok 👍

Reply:
Python 3.11 was in fact generating random order and we were running tests in python 3.8 with fixed seed to prevent that. The rust implementation guarantees ordering, not very important, but simplifies testing.